spec/dor/identifiable_spec.rb

```diff
@@ -8,6 +8,18 @@ class IdentifiableItem < ActiveFedora::Base
   before(:each) { stub_config }
   after(:each)  { unstub_config }
 
+  before(:all) do
+    @mock_druid = 'druid:fg890hi1235'
```

Comment:
i'd be inclined to call it `@mock_rel_druid` since it's acting as both the mock apo druid and the mock collection druid.

also, as a personal preference, for something like this, i prefer to use an id that reads as obviously fake or bad (e.g. `druid:badRel` or `druid:zy987xw6543`, or whatever).  this one differs from the old `mock_apo_druid` by one character, and i'm not sure if that was meant to act as a "real" druid either).